normits_demand/models/notem/production_models.py

```diff
@@ -198,6 +175,32 @@ def __init__(self,
             instantiate_msg="Initialised HB Production Model",
         )
 
+        self.zoning = nd.get_zoning_system(zoning_name)
+
+        # Define required columns
+        self._target_col_dtypes = {
+            'pop': {
+                self.zoning.col_name: self.zoning.unique_zones.dtype,
+                'area_type': int,
+                'tfn_traveller_type': int,
+                'people': float
+            },
+            'trip_rate': {
+                'tfn_tt': int,
+                'tfn_at': int,
+                'p': int,
+                'trip_rate': float
+            },
+            'm_tp': {
+                'p': int,
+                'tfn_tt': int,
+                'tfn_at': int,
+                'm': int,
+                'tp': int,
+                'split': float
+            },
+        }
+
```

Comment:
This is quite a large definition. It's probably better to call a function that returns a dictionary when handed the zoning system.

We want to keep the `__init__` as simple as possble!